docs/dev/fundamentals/testnet.md

```diff
@@ -29,9 +29,9 @@ All the existing SDKs for Ethereum will work out of the box and your users will
 
 The only other place where using zkSync SDK is required is during contract deployment. This can be easily done through our hardhat plugin.
 
-## Hello World on zkSync
+## Quickstart tutorial on zkSync
 
-Check out our step-by-step [tutorial](../developer-guides/hello-world.md), where you will learn:
+Check out our step-by-step [quickstart tutorial](../developer-guides/hello-world.md), where you will learn:
```

Comment:
Same reason as the one above.
We can say "Check out our [quickstart guide](../developer-guides/hello-world.md), where you will learn:"

Reply:
fixed